sources/engine/Stride.Assets/SpriteFont/Compiler/SignedDistanceFieldFontImporter.cs

```diff
@@ -129,7 +129,7 @@ public void Import(SpriteFontAsset options, List<char> characters)
                 return;
 
             // Get the msdfgen.exe location
-            var msdfgen = ToolLocator.LocateTool("msdfgen.exe") ?? throw new AssetException("Failed to compile a font asset, msdfgen was not found.");
+            var msdfgen = ToolLocator.LocateTool("msdfgen") ?? throw new AssetException("Failed to compile a font asset, msdfgen was not found.");
 
             msdfgenExe = msdfgen.FullPath;
```

Comment:
Probably need to do `msdfgenExe = msdfgen.ToOSPath()` here.

Reply:
@Kryptos-FR And I'm sorry I forgot to reply, but here you go :)

Reply:
That's a mistake, I couldn't locally compile Stride because of that.

Reply:
Alright, I'll try to make a fix for that :)